### ISSUE TITLE
Background changes when more than a couple files are opened

### ISSUE THREAD

**Issue by nobleach** [marko-cerovac/material.nvim#35]
I am using the oceanic variant. When I open my file explorer (nvim-tree.lua), the backgrounds are different (which is great!):
![image](https://user-images.githubusercontent.com/217134/119739280-3537d280-be3f-11eb-974f-f0cf0bd6a905.png)

If I open a few file in other tabs, all of a sudden the background of those buffers matches the file explorer's background:
![image](https://user-images.githubusercontent.com/217134/119739369-5c8e9f80-be3f-11eb-8349-99ee6c3ecf98.png)

The best I can tell, it's when I open a file from another folder. Yet, any other file I open afterward has this background.

Also, after pasting this screenshot, I just noticed the line numbers have disappeared as well.

**Comment by marko-cerovac** [comment]
That's really weird. I've tried to replicate the issue and I have succeeded. I'm pretty sure that this is an issue with the latest version of nvim-tree since it doesn't happen with the with the older one. Also, I've noticed that one of the latest commits to nvim-tree.lua changed tabing. I think the reason that there are no numbers is that the new buffer that you open inherits the layout of nvim-tree, instead of the normal layout that it should. I'll work on this in the next few days, and see if I can find a workaround, but I think it's up to the maintainers of nvim-tree.lua
